### DISCUSSION TITLE
How to delete conversion + original when deleting a media item?

### DISCUSSION THREAD

**Discussion by BenjaminVerkleij** [spatie/laravel-medialibrary#2009] [Q&A]
If you've attached a media item using a separate disk for the conversions with storingConversionsOnDisk() the original file isn't deleted from the disk when you delete said media item.

If, for example, I delete a media item with $mediaItems[0]->delete(); all conversions in the conversions_disk directory are correctly deleted but the original file in the main media disk is not deleted.

Is it possible to delete both automatically?

I'm using Media Library 8.5.0.

**Comment by freekmurze** [comment]
I'm thinking the medialibrary does delete the original item as well. 

Could you PR a failing test that proves that this behaviour is broken?

**Comment by BenjaminVerkleij** [comment]
I'll try to look into this asap.

I've noticed that php artisan medialibrary:clean does remove the original when left behind on deleting the media item itself.
